Tools/MatrixPilot-SIL/UDBSocket.h

```diff
@@ -9,10 +9,11 @@
 #ifndef MatrixPilot_SIL_SIL_sockets_h
 #define MatrixPilot_SIL_SIL_sockets_h
 
-#if (!defined(WIN32) || defined(WIN))
+#if (!defined(WIN32) || defined(WIN) || !defined(_MSC_VER))
 #include <stdint.h>
 #else
-#include "stdint-win.h"
+//#include "stdint-win.h" // only for MS Visual C++ builds

```

Comment:
I left this one here as for the moment I'm without a Windows development machine, and hence have not done any test builds with VC++ for quite some time. When it is tested, should something WRT to this be astray, this 'comment' may add debugging value.